mappings/net/minecraft/world/gen/MultiNoiseParameters.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_2088 net/minecraft/world/gen/MultiNoiseParameters
+	FIELD field_24727 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_34626 temperature Lnet/minecraft/class_5216$class_5487;
+	FIELD field_34627 humidity Lnet/minecraft/class_5216$class_5487;
+	FIELD field_34628 continentalness Lnet/minecraft/class_5216$class_5487;
+	FIELD field_34629 erosion Lnet/minecraft/class_5216$class_5487;
+	FIELD field_34630 weirdness Lnet/minecraft/class_5216$class_5487;
+	FIELD field_34631 shift Lnet/minecraft/class_5216$class_5487;
```

Comment:
I recall this was named `offset` before as this is a penalty on the biome (to make it more rare). What's the reasoning of `shift`, just curious?

Reply:
Offset still exists. This is the noise that was previously called "location offset", but since the name is now in the codec, I renamed it to that.